src/UWP/ArcGIS.UWP.Viewer/Samples/Geometry/SnapGeometryEdits/SnapGeometryEdits.xaml.cs

```diff
@@ -52,6 +52,7 @@ public partial class SnapGeometryEdits
         public SnapGeometryEdits()
         {
             InitializeComponent();
+            SnappingControls.Visibility = Visibility.Collapsed;
```

Comment:
Small improvement: Collapsing in constructor instead of in XAML makes the sample UI visible/editable in VisualStudio's XAML designer. Without this change, you can't see any controls in the designer.